dts/arm/nxp/nxp_lpc55S6x.dtsi

```diff
@@ -43,7 +43,7 @@
 		reg = <0x30030000 DT_SIZE_K(64)>;
 	};
 
-	sramx:memory@50000000{
+	sramx:memory@40000000{
```

Comment:
Neither of these values are right. sramx is at 0x0400_0000 non-secure, 0x1400_0000 secure.

Reply:
Changed to `0x1400_0000` since the other addresses are in secure space.